www/src/css/_animations.scss

```diff
@@ -0,0 +1,8 @@
+@use './var' as *;
+
+@mixin subtle-border-animation {
+  &:hover {
+    border-color: transparent; /* remove the border's colour */
```

Comment:
If you’re animating from `#666` to transparent, you’ll want to use `rgba(#666, 0)` instead, otherwise your animation will transition to the wrong color as it fades out.

You might not notice it in this animation, but if you were using a really bold color it’d be noticably turning black as it disappeared